src/cascadia/TerminalSettingsModel/Command.cpp

```diff
@@ -178,6 +179,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         }
 
         JsonUtils::GetValueForKey(json, IconKey, result->_IconPath);
+        JsonUtils::GetValueForKey(json, KeysKey, result->_Keys);

```

Comment:
Looks like I still need some way to check for `SettingsLoadWarnings::TooManyKeysForChord`

Reply:
haha nope. This still appears. KeyMap probably runs into this and throws it in. Resolving.